packages/utils/src/network/addrs.ts

```diff
@@ -125,6 +123,47 @@ export function ipToU8aAddress(address: string, family: NetworkInterfaceInfo['fa
   }
 }
 
+/**
+ * Returns the prefix length of a network prefix
+ * @param prefix network prefix, e.g. `new Uint8Array([255,255,255,0])`
+ * @returns the prefix length, e.g. 24
+ */
+export function prefixLength(prefix: Uint8Array) {
+  const masks: number[] = [128, 192, 224, 240, 248, 252, 254, 255]
```

Comment:
Why not adding also `0` in this array and then have another `numbits` array which contains [0,1,2,...,8].
Then you can have just one `for` loop which compares with the `masks` array and accumulate a value from `numbits` at the same index. This will save the inner `for` loop below.

Reply:
Added some modifications